src/iarc7_motion/iarc_tasks/track_roomba_task.py

```diff
@@ -37,8 +37,9 @@ class TrackRoombaTask(object, AbstractTask):
     def __init__(self, task_request):
 
         self._roomba_id = task_request.frame_id  + '/base_link'
-        # true means hit roomba, false means block
-        self._tracking_mode = task_request.tracking_mode
+        self._time_to_track = task_request.time_to_track
+        self._x_overshoot = task_request.x_overshoot
+        self._y_overshoot = task_request.y_overshoot
```

Comment:
Can we check that these overshoots are possible given the limitations imposed by _MAX_START_TASK_DIST. The check would need to be below line 83. Throw a valueerror exception if it occurs.

Reply:
Fixed